docs/docsite/rst/userguide/projects.rst

```diff
@@ -472,14 +505,17 @@ You can create different repos with different namespaces/collections in them. Bu
 6. If the |ah| has self-signed certificates, click the toggle to enable the setting **Ignore Ansible Galaxy SSL Certificate Verification**. For **public Automation Hub**, which uses a signed certificate, click the toggle to disable it instead. Note this is a global setting:
 
 .. image:: ../common/images/settings-jobs-ignore-galaxy-certs.png
+   :alt: setting of jobs ignore galaxy certs
```

Comment:
May I suggest a more descriptive text here, like:
"Job Settings page showing where to enable the option to ignore Ansible Galaxy SSL Certificate Verification."